### PR TITLE
Adds Account Management API

### PR DESCRIPTION
**Pull request by angrybear11**

### CODE REVIEW

File: lib/paysafe/account_management_service.rb [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+      uri: "/accountmanagement/monitor",
+      )
+
+      response = @client.process_request request
+      return response.status == "READY"
+    end
+
+    def createMerchantEntity merchantEntity
+      request = Request.new(
+      method: Request::POST,
```
Comment (zggoldman) [inline lib/paysafe/account_management_service.rb]: I realize we're matching the existing style, but I still feel like I have to mention how awful this indentation pattern is.

File: lib/paysafe/account_management_service.rb [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+      )
+
+      response = @client.process_request request
+      AccountManagement::MerchantEntity::new response
+    end
+
+    def createMerchantAccount merchantAccount
+      request = Request.new(
+      method: Request::POST,
+      uri: prepare_uri("/merchants/#{merchantAccount.merchantId}/accounts"),#?operationMode=consolidated
```
Comment (zggoldman) [inline lib/paysafe/account_management_service.rb]: Do we need this trailing comment?

File: lib/paysafe/account_management_service.rb [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+      response = @client.process_request request
+      AccountManagement::MerchantAccountAddress::new response
+    end
+
+    def createBusinessOwner businessOwner
+      request = Request.new(
+      method: Request::POST,
+      uri: prepare_uri("/accounts/#{businessOwner.accountId}/businessowners"),#?operationMode=consolidated
+      body: businessOwner.get(
+        required = ['firstName', 'lastName', 'jobTitle', 'phone', 'dateOfBirth', 'ssn', 'isApplicant', 'currentAddress']
```
Comment (zggoldman) [inline lib/paysafe/account_management_service.rb]: Line length?

File: lib/paysafe/account_management_service.rb [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+
+    def submitMerchantAccountForActivation accountId
+      request = Request.new(
+      method: Request::POST,
+      uri: prepare_uri("/accounts/#{accountId}/activation"),
+      body: {}
+      )
+
+      response = @client.process_request request
+      bindings.pry
```
Comment (zggoldman) [inline lib/paysafe/account_management_service.rb]: Debugger

File: lib/paysafe/account_management_service.rb [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+      response
+      # response.account.status
```
Comment (zggoldman) [inline lib/paysafe/account_management_service.rb]: Commented code

File: lib/paysafe/account_management_service.rb [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+      )
+
+      response = @client.process_request request
+      AccountManagement::MerchantAccountAddress::new response
+    end
+
+    def createBusinessOwner businessOwner
+      request = Request.new(
+      method: Request::POST,
+      uri: prepare_uri("/accounts/#{businessOwner.accountId}/businessowners"),#?operationMode=consolidated
```
Comment (zggoldman) [inline lib/paysafe/account_management_service.rb]: Do we need the trailing comment.

File: lib/paysafe/account_management_service.rb (hunk 1 above)
Comment (angrybear11) [reply lib/paysafe/account_management_service.rb]: I looked again and the existing style is inconsistent so I'll do some cleanup.

File: lib/paysafe/account_management_service.rb [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+    def submitMerchantAccountForActivation accountId
+      request = Request.new(
+        method: Request::POST,
+        uri: prepare_uri("/accounts/#{accountId}/activation"),
+        body: {}
+      )
+
+      response = @client.process_request request
+      response
+      # response.account.status
```
Comment (zggoldman) [inline lib/paysafe/account_management_service.rb]: Still some commented code here. Can we remove it?

**Pull request merged**